frontend/src/app/plot/plot.component.ts

```diff
@@ -38,32 +40,62 @@ export class PlotComponent {
    *    y: Takes an array of y-axis values.
    *    (Note: The lengths of x and y must be equal.)
    *    type: How the data is displayed to the user. 'scattergl' and 'historgram'
-   *      will most likely be the only types used. (Avoid 'scatter' type since it
+   *      are the only currently used types. (Avoid 'scatter' type since it
    *      has poorer performance on large datasets than 'scattergl'.)
    *    mode: Finer details for how to display the type. 'markers' displays only a dot
    *      for each datapoint. 'lines' displays a line through all datapoints. 'lines+markers'
    *      displays both.
+   *    marker: A map that allows custom changes to markers.
+   *      marker.symbol: The shape markers take. Options used: 'circle', 'diamond', 'star'.
+   *      https://plotly.com/python/marker-style/#custom-marker-symbols
+   *    id: The id used by Sensor Visualizer to uniquely identify traces so they can be
+   *      removed, altered, or referenced in some way.
+   *    visible: If true, the trace will show up. If false, the trace is invisible.
+   *    name: The trace name displayed to the user.
+   *    xbins: The size of the buckets in the histogram if type === 'histogram' TODO
    */
   plot_data = [
     {
       x: [0],
       y: [0],
       type: 'scattergl',
       mode: 'markers',
+      marker: {symbol: 'diamond'},
       id: 0,
       visible: true,
       name: 'Placeholder Point',
+      xbins: null,
     },
   ];
 
-  // Plot Configurations.
-  plot_layout = {title: 'Add a new dataset.', legend: 'false'};
+  /**
+   * Plot configurations.
+   * hovermode must be set to closest. When it is set to closest, only
```

Comment:
These are contradictory
"hovermode must be set to closest." & "When it is set to closest"